src/qt3utils/nidaq/customcontrollers.py

```diff
@@ -0,0 +1,84 @@
+import nidaqmx
+import nipiezojenapy
+from typing import List
+import time
+import logging
+
+logger = logging.getLogger(__name__)
+class VControl(nipiezojenapy.BaseControl):
```

Comment:
what's the benefit / reason to subclass from nipiezojenapy.BaseControl?

Reply:
I think we shouldn't subclass. There are a number of reasons if you look into the details. BaseControl assumes we're working in a 3-dimensional space whereas VControl is for one dimension.